### PR TITLE
:book: comparison with RPC added

### PR DESCRIPTION
**Pull request by ghost**
RPC vs RESTAPI/OPENAP task

### CODE REVIEW

**Review by yelizariev — CHANGES_REQUESTED** [review]
Еще преимущство в том, что куча утилит поддерживают OpenAPI

File: openapi/static/description/index.html [hunk 1]
```diff
@@ -255,6 +255,36 @@ <h2 class="oe_slogan">Choose a platform that fit your needs</h2>
     </div>
 </section>
 
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+
+            <div class="alert alert-success oe_mt32" style="padding:0.3em 0.6em; font-size: 150%;">
+                <i class="fa fa-linode" aria-hidden="true"></i><b> RESTAPI/OPENAPI vs RPC </b>
+                <ul class="list-unstyled"><i>Integrated RPC functionality differs from Our implementation of RESTAPI/OPENAPI</i>:
```
Comment (ilmir-k) [inline openapi/static/description/index.html]: Our implementation of RESTAPI/OPENAPI has the following advantages over the built-in RPC functionality

File: openapi/static/description/index.html [hunk 2]
```diff
@@ -255,6 +255,36 @@ <h2 class="oe_slogan">Choose a platform that fit your needs</h2>
...
+                    <li>
+                        <i class="fa fa-check-square-o text-primary"></i>
+                        You can restrict access to the specific models, to the function call
+                    </li>
+
+                    <li>
+                        <i class="fa fa-check-square-o text-primary"></i>
+                        You can authorizate through tokens (without passwords)
```
Comment (ilmir-k) [inline openapi/static/description/index.html]: ``make authorization``

File: openapi/static/description/index.html [hunk 3]
```diff
@@ -255,6 +255,36 @@ <h2 class="oe_slogan">Choose a platform that fit your needs</h2>
...
+                    </li>
+
+                    <li>
+                        <i class="fa fa-check-square-o text-primary"></i>
+                        You can make requests with the one request <p>
+                        (<i>for example</i>, in RPC you need to Login first, then get uid...)
```
Comment (ilmir-k) [inline openapi/static/description/index.html]: ... you need to be authorized first ...

**Comment by ilmir-k** [comment]
@yelizariev you had requested changes here

**Comment by ilmir-k** [comment]
@Mirgalimova

**Review by yelizariev — APPROVED** [review]

**Comment by ilmir-k** [comment]
I approve to merge it now

**Pull request merged**